fractal_server/app/routes/api/v1/project.py

```diff
@@ -85,12 +85,11 @@ async def create_project(
     )
 
     db_project = Project(**project.dict())
-    db_project.user_list.append(user)
+
     try:
         db.add(db_project)
         await db.commit()
         await db.refresh(db_project)
-        await db.close()
     except IntegrityError as e:
```

Comment:
I don't think we use this pattern (catching `IntegrityError` and transforming it from 500 to 422) often - to be checked. We can probably drop it from this endpoint.

Let's make these 500 errors evident, and then if it's relevant we should turn this into a helper function and use it everywhere to wrap the `db.commit` statements from endpoints.

Reply:
We catch `IntegrityError`s just in `app/routes/api/v1/project.py::create_project` and `app/security/__init__.py::_create_first_user`.

Reply:
Closed with https://github.com/fractal-analytics-platform/fractal-server/pull/1530